### PR TITLE
Fix - Fail when using empty string env vars 

### PR DESCRIPTION
**Pull request by roni-frantchi**
Passing an environment variable with empty string such as `FOO=` causes the CLI to fail on validation.  
This PR fixes that.

### CODE REVIEW

File: node/env0-deploy-cli.js [hunk 1]
```diff
@@ -78,8 +78,8 @@ const getEnvironmentVariablesOptions = (environmentVariables) => {
   if (environmentVariables && environmentVariables.length > 0) {
     console.log('getting Environment Variables from options:', environmentVariables);
     environmentVariables.forEach(config => {
-      const configArray = config.split(/=(.+)/);
-      if (configArray.length === 3) {
+      const configArray = config.split('=');
```
Comment (omryhay1) [inline node/env0-deploy-cli.js]: This is going to be problematic when the value contains a '=' for example it is a base64: 
`b21yeSBoYXkgCg==`

File: node/env0-deploy-cli.js [hunk 2]
```diff
@@ -78,16 +78,9 @@ const getEnvironmentVariablesOptions = (environmentVariables) => {
...
-      if (configArray.length === 3) {
-        result.push({
-          name: configArray[0],
-          value: configArray[1]
-        })
-      }
-      else {
-        throw new Error(`environmentVariables options are invalid`);
-      }
+      let [name, ...value] = config.split('=');
```
Comment (omryhay1) [inline node/env0-deploy-cli.js]: @Roni I know it's mainly my code, but could you please add some unit tests here with the use case of an empty environment variable and case that there is a '=' in the value?

Comment (roni-frantchi) [reply node/env0-deploy-cli.js]: A proper testbed for this module is missing.  
We'll need to do some refactoring here to extract a cli module exposing this util as right now this code is deeply tangled in the cli code.  
I think that's out of scope right now.
